packages/typespec-ts/test/modularUnit/enumUnion.spec.ts

```diff
@@ -621,7 +621,10 @@ describe("header parameters", () => {
       const schemaOutput = await emitModularModelsFromTypeSpec(
         tspDefinition,
         false,
-        true
+        true,
+        false,
+        false,
+        false
```

Comment:
I awalys want to do refactoring these options to option bag, otherwise we should be carefully with the order.